src/config/handlers.ts

```diff
@@ -0,0 +1,84 @@
+import yargs from 'yargs';
+import { configSchema } from './schemas';
+import { Config } from '../types';
+import fs from 'fs';
+import { hideBin } from 'yargs/helpers';
+import _ from 'lodash';
+import { defaultConfig } from '../constants';
```

Comment:
I'd sort these imports like this:
```suggestion
import _ from 'lodash';
import fs from 'fs';
import yargs from 'yargs';
import { hideBin } from 'yargs/helpers';
import { Config } from '../types';
import { configSchema } from './schemas';
import { defaultConfig } from '../constants';
```
So that we have external dependencies, then local dependencies